specification/dartLangSpec.tex

```diff
@@ -8653,48 +8659,20 @@ \subsection{Constants}
   is a potentially constant expression.
 
 \item
-  A constant object expression (\ref{const}),
-  \code{\CONST{} $C$<$T_1,\ \ldots,\ T_k$>(\metavar{arguments})} or
-  \code{\CONST{} $C$<$T_1,\ \ldots,\ T_k$>.\id(\metavar{arguments})},
-  or either expression without the leading \CONST{} that occurs in
-  a constant context,
-  is a potentially constant expression.
-  It is further a constant expression if the invocation evaluates to an object.
-  % \ref{const} requires each actual argument to be a constant expression,
-  % but here we also catch errors during evaluation, e.g., `C(1, 0)` where
-  % `C(double x, double y): z = x / y;`.
-  It is a compile-time error if a constant object expression is
-  not a constant expression (\ref{const}).
+  A constant object expression (\ref{const})
+  is a potentially constant and constant expression.
```

Comment:
This doesn't remove the requirement that subexpressions and type arguments are constant expressions/types, because that's written in the "constant constructor invocation".

Should we say this here, as commentary?